hydra-plutus/src/Hydra/Contract/Commit.hs

```diff
@@ -32,19 +33,28 @@ import PlutusTx (CompiledCode, toBuiltinData)
 import qualified PlutusTx
 import qualified Prelude
 
-data CommitRedeemer = CollectCom | Abort
+data CommitRedeemer
+  = ViaCollectCom
+  | ViaAbort
```

Comment:
What's the rationale of these new names?

Reply:
Conflicting names. I thought this was even better now anyway :p

Reply:
I do not think it's better, but I guess that's subjective. I wonder what conflicted? We do explicit imports and for these modules even qualified ones (e.g. to get `Commit.DatumType`) and the new names are also shared between initial and commit validators.